charts/gsp-cluster/templates/02-gsp-system/service-operator/deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
         - --cluster={{ .Values.global.cluster.name }}
         - --kiam-server-role-arn={{ .Values.serviceOperator.kiamServerRoleARN }}
         - --role-permissions-boundary-arn={{ .Values.serviceOperator.permissionsBoundaryARN }}
+        - --rds-from-worker-security-group={{ .Values.rds-from-worker-security-group }}
```

Comment:
You shouldn't use hyphens in yaml map keys

`rds-from-worker-security-group` should be `rdsFromWorkerSecurityGroup`

I'm surprised helm didn't complain...